src/virt-install

```diff
@@ -113,6 +113,20 @@ elif args.kickstart:
 else:
     raise SystemExit("Either --kickstart or --image-conf must be specified")
 
+# To support different distro builds using Fedora anaconda,
+# installclass must be explicit for anaconda to find the correct directory.
+try:
+    subprocess.check_call(['ostree', f"--repo={args.ostree_repo}", 'ls', f"{args.ostree_ref}", "/usr/lib/ostree-boot/efi/EFI/redhat"])
```

Comment:
`f"{args.ostree_ref}"` is just `args.ostree_ref`

Reply:
I'd probably do
`if subprocess.call(..., stderr=subprocess.DEVNULL) == 0:`

Then you can drop the `except`

Reply:
(Otherwise LGTM!)